src/docs/sphinx/dev_guide/mfem_dev.rst

```diff
@@ -0,0 +1,49 @@
+.. ## Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+.. ## other Serac Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+======================
+MFEM development build
+======================
+
+Streamlining back-porting features to MFEM
+------------------------------------------
+
+Occasionally, serac developers may want to back-port useful features back to MFEM. MFEM has two separate build systems (`make` and `cmake`). The default `make` system that is suggested requires installing suggested packages. Fortunately, mfem's `cmake` build system is more compatible with serac's cmake-based build system. Since serac provides tools to build mfem's dependencies, one can streamline the back-porting process by re-using serac's existing hostconfig files located in `<SERAC_ROOT_DIR>/host-configs`.
```

Comment:
Can you line wrap this and capitalize Serac, MFEM, and CMake where it makes sense?

"hostconfig" -> "host-config"